config/sync/image.style.spotlight_image_370x370_.yml

```diff
@@ -0,0 +1,15 @@
+uuid: 1b966590-30b9-4287-bc8d-4bf1945493aa
+langcode: en
+status: true
+dependencies: {  }
+name: spotlight_image_370x370_
+label: 'Spotlight Image (370x370)'
+effects:
+  5611bc53-b983-4421-88a7-f9b814aec0da:
+    uuid: 5611bc53-b983-4421-88a7-f9b814aec0da
+    id: image_scale_and_crop
```

Comment:
Should use the focal point scale and crop

Reply:
Spotlight image has been updated from `scale and crop` to `focal point scale and crop` 👍